src/features/channels/components/channel-list/search-channel/SearchChannel.tsx

```diff
@@ -85,15 +90,15 @@ const ChannelList = () => {
 
             <ListItem disablePadding>
               <ListItemButton>
-                <ListItemIcon style={{ color: 'gray' }}>
+                <ListItemIcon style={{ color: '#CFF4D2' }}>
                   <SettingsIcon />
                 </ListItemIcon>
                 <ListItemText primary="Settings" />
               </ListItemButton>
             </ListItem>
             <ListItem disablePadding>
               <ListItemButton>
-                <ListItemIcon style={{ color: 'blue' }}>
+                <ListItemIcon style={{ color: '#CFF4D2' }}>
```

Comment:
Maybe consider adding this color to the theme

Reply:
I will change to that color then.